### DISCUSSION TITLE
unable to download videos directly in Google Chrome

### DISCUSSION THREAD

**Discussion by Jusiu** [aclap-dev/video-downloadhelper#596] [Help and Q&A]
### Your question

I have found that I am unable to download videos directly in Google Chrome. :) 
chrome verison : 125.0.6422.113  (arm64)
video downloadhelper Version 9.0.1.13 (stable), target google, locale zh-TW.
macOS verison : sonoma 14.5

### Addon & CoApp version and details

I have found that I am unable to download videos directly in Google Chrome. :) 
chrome verison : 125.0.6422.113  (arm64)
video downloadhelper Version 9.0.1.13 (stable), target google, locale zh-TW.
macOS verison : sonoma 14.5

**Comment by paulrouget** [comment]
Can you give more details. Please follow these instructions: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F
